frappe/exceptions.py

```diff
@@ -271,3 +271,11 @@ class InvalidDatabaseFile(ValidationError):
 
 class ExecutableNotFound(FileNotFoundError):
 	pass
+
+
+class LinkExpiredError(ValidationError):
+	pass
```

Comment:
```suggestion
class LinkExpired(ValidationError):
	pass
```
Should I remove "Error" from the class name? I think if link expiry is classified as "Exception" and not "Error"?

Reply:
Yes, we can learn from the convention used in `SessionStopped` or `SessionExpired` and avoid suffixing with `Error`.

`InvalidKey`, on the other hand could be suffixed with `Error` (see `InvalidEmailAddressError`, `InvalidNameError`). But we also have many `Invalid...` classes without that suffix. I think it's either is fine.